Order external urls

marusak opened the pull request:
Fixes abrt/faf#598

mkutlak commented on src/webfaf/templates/_helpers.html:
```diff
@@ -371,11 +371,14 @@
+        </a>
+      </li>
+    {% endfor %}
+    <li class="empty-line"></li>
```
Maybe do something like this:
\<li class="empty-line"\> \<hr /\>  \</li\>

li.empty-line hr {margin: 0.5em 0;} 
li.empty-line:last-child { display: none; }

marusak replied:
hmm, this adds line. It stretches across the whole available space. I don't this it looks good. But I let it for others to decide. Ideas anyone?
![after](https://user-images.githubusercontent.com/12330670/33660046-d2ef63fc-da82-11e7-80ea-730cd4472e45.png)

mkutlak replied:
Yeah, it looks weird when the URLs are short. This is how it looks like with normal URLs. ![fafpr](https://user-images.githubusercontent.com/24898105/33662907-f6ebbcfa-da8d-11e7-820f-d279c89b7a7f.png)

marusak commented:
Updated. The line is now 50% of the width. And I also added tooltips. 
![report](https://user-images.githubusercontent.com/12330670/33831641-2375d488-de79-11e7-8a38-fc283149b835.png)
![problem_full](https://user-images.githubusercontent.com/12330670/33831642-2395158c-de79-11e7-8734-cf7d244a1956.png)

mkutlak commented:
The tooltip could say something like this:
"The URLs are grouped by reports. Newest URL is always at the top."
Nothing else comes to my mind.

Otherwise it looks good and can be merged.

marusak commented:
Updated.

mkutlak commented:
LGTM. Thank you!
